server.js

```diff
@@ -214,20 +233,57 @@ async function work(body) {
     );
   }
 
-  github.issues.createComment({
-    user: data.repository.owner.login, // 'fbsamples'
-    repo: data.repository.name, // 'bot-testing'
-    number: data.pull_request.number, // 23
-    body: message
-  });
-
-  if (repoConfig.assignToReviewer) {
-    github.issues.edit({
+  function createComment(data, message, resolve, reject) {
+    github.issues.createComment({
       user: data.repository.owner.login, // 'fbsamples'
       repo: data.repository.name, // 'bot-testing'
       number: data.pull_request.number, // 23
-      assignee: reviewers[0],
+      body: message
+    }, function(err, result) {
+      if(err){
+        if(typeof reject === 'function') return reject(err)
+      }
+    })
+  }
+
+  function assignReviewer(data, reviewers, resolve, reject) {
+    if (repoConfig.assignToReviewer) {
```

Comment:
I like using early returns:

```js
if (!repoConfig.assignToReviewer) {
  return;
}
...
```

this way, you have one less level of indentation